_includes/selector.html

```diff
@@ -512,9 +527,9 @@
                 }
 
                 // This should be unnecessary and removed after 23.06
-                if (isArm) {
-                    cupy_inst = "\npip install cupy-cuda11x -f https://pip.cupy.dev/aarch64";
-                }
+                // if (isArm) {
+                //     cupy_inst = "\npip install cupy-cuda11x -f https://pip.cupy.dev/aarch64";
+                // }
```

Comment:
any reason we can't just delete these lines altogether?

Reply:
I think we don't need this anymore -- cupy is on pip for both AMD64 and ARM now. Safe to delete.

Reply:
Yeah sorry, meant to ask if they were safe. Deleting now.